src/view/com/notifications/Feed.tsx

```diff
@@ -51,6 +60,8 @@ export function Feed({
   const {_} = useLingui()
   const moderationOpts = useModerationOpts()
   const {checkUnread} = useUnreadNotificationsApi()
+  const [tab, setTab] = React.useState<(typeof tabs)[number]>(tabs[0])
```

Comment:
Oooo tab switching can be instant with:

```js
const notificationTypes = React.useDeferredValue(tab.types)
```

But the UI needs an indicator that the render is behind...